docs/cloud-native-application-developers-manual.md

```diff
@@ -0,0 +1,60 @@
+# Cloud Native Application Developers Manual
```

Comment:
"cloud-native" is the usual term (hyphenated)

Reply:
After reading this, I wonder if this title is an accurate representation of the content. I understand "manual" to be a fairly exhaustive document covering all aspects. This seems to be more like an "overview" or "introductory concepts" document. I think it is probably more useful than a manual, so perhaps consider a different title eg. "Key Concepts", "Quickstart", "Overview", "Quick Reference" etc.

Reply:
Makes sense. Renaming it to "Application Developer's Reference Guide"